services-geojson/src/commonMain/kotlin/org/maplibre/geojson/utils/Json.kt

```diff
@@ -0,0 +1,21 @@
+package org.maplibre.geojson.utils
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.ClassDiscriminatorMode
+import kotlinx.serialization.json.Json
+
+/**
+ * Internal JSON instance used for serialization and deserialization of GeoJSON models
+ */
+@OptIn(ExperimentalSerializationApi::class)
+internal val json = Json {
+    classDiscriminatorMode = ClassDiscriminatorMode.ALL_JSON_OBJECTS
+
+    // Encode
+    encodeDefaults = true
```

Comment:
wouldn't that encode e.g. `null` for every `Point` (and other `Geometry`) for `bbox`?